pkg/router/template/template_helper.go

```diff
@@ -179,8 +195,9 @@ var helperFunctions = template.FuncMap{
 	"matchValues":  matchValues,  //compares a given string to a list of allowed strings
 
 	"genSubdomainWildcardRegexp": genSubdomainWildcardRegexp, //generates a regular expression matching the subdomain for hosts (and paths) with a wildcard policy
-	"generateRouteRegexp":        generateRouteRegexp,        //generates a regular expression matching the route hosts (and paths)
+	"generateRouteRegexp":        GenerateRouteRegexp,        //generates a regular expression matching the route hosts (and paths)
```

Comment:
Why is this exported?

Reply:
If you are trying to reuse this, put it in its own package (`templatehelpers` maybe) and include it here.  That gets more crap out of this package.

Reply:
ok will do.